README.md

```diff
@@ -1,14 +1,9 @@
-# egress-assess-packer 💀📦 #
+# egress-assess-packer #
 
 [![GitHub Build Status](https://github.com/cisagov/egress-assess-packer/workflows/build/badge.svg)](https://github.com/cisagov/egress-assess-packer/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) GitHub
-[Packer](https://packer.io) project started. This skeleton project
-contains [licensing information](LICENSE), as well as
-[pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
+This project can be used to build a Debian AMI with
+[Egress-Assess](https://github.com/FortyNorthSecurity/Egress-Assess) installed.
```

Comment:
You're installing from [tarrell13/Egress-Assess](https://github.com/tarrell13/Egress-Assess), and according to #13 you intend to continue doing so, so this link should be updated:
```suggestion
[Egress-Assess](https://github.com/tarrell13/Egress-Assess) installed.
```